src/App.tsx

```diff
@@ -1,24 +1,95 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Header } from './components/header/Header';
+import { List } from './components/list/List';
+import { Footer } from './components/footer/Footer';
 import { UserWarning } from './UserWarning';
+import { Warnings } from './components/warnings/Warnings';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Select } from './types/Select';
+import { Error } from './types/Error';
 
-const USER_ID = 0;
+const USER_ID = 10514;
+
+const { All, Active, Completed } = Select;
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<[] | Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(All);
+  const [error, setError] = useState('');
+  const [processings, setProcessings] = useState<number[] | []>([]);
+
+  const loadTodos = () => {
+    getTodos(USER_ID)
+      .then((todos) => setTodoList(todos))
+      .catch(() => setError(Error.Get));
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
+  useEffect(() => {
+    setTimeout(() => setError(''), 3000);
+  }, [error]);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  function filterList() {
+    switch (filterBy) {
+      case Active:
+        return todoList.filter((todo: Todo) => !todo.completed);
+
+      case Completed:
+        return todoList
+          .filter((todo: Todo) => todo.completed || todo.completed === null);
```

Comment:
Consider checking for 'todo.completed === false' instead of 'todo.completed === null', as the 'completed' property should be a boolean.